src/components/Calculator.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import './Calculator.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+ handleChange = () => {
+   // this.setState({ display: event.target.value });
+ }
+
+ render() {
+   return (
+     <div className="Calculator">
+       <table>
+         <tr className="table-row">
+           <td colSpan="4">
+             <input
+               type="text"
+               name="display"
+               // value={this.state.display}
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻

Reply:
Thanks @BenMukebo